AccordionItem.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import styles from "./Faq.module.scss"
+
+const AccordionItem = ({ children, title }) => {
+  const [is_content_visible, setContentVisible] = React.useState(false)
+  const togglePanel = () => {
+    setContentVisible(!is_content_visible)
+  }
+  return (
+    <div className={styles["accordion-wrapper"]}>
```

Comment:
Same here, change the class name to accordionWrapper and call it as `styles.accordionWrapper`. Try to do this for all css styles you try to call with brackets []